iocBoot/iocBPMRFFEApp/save_restore.cmd

```diff
@@ -0,0 +1,30 @@
+# Save and restore setup
+
+save_restoreSet_status_prefix("${P}${R}")
+# Debug-output level
+save_restoreSet_Debug(0)
+
+# Is it okay to save/restore sets with missing values (no CA connection to PV)?
+save_restoreSet_IncompleteSetsOk(1)
+# Save dated backup files?
+save_restoreSet_DatedBackupFiles(1)
+
+# Number of sequenced backup files to write
+save_restoreSet_NumSeqFiles(3)
+# Time interval between sequenced backups
+save_restoreSet_SeqPeriodInSeconds(300)
+
+# Specify where save files should be
+set_savefile_path("/opt/autosave", "rffe")
+
+# Specify what save files should be restored.  Note these files must be
+# in the directory specified in set_savefile_path(), or, if that function
+# has not been called, from the directory current when iocInit is invoked
+set_pass0_restoreFile("${AUTOSAVE_FILENAME}.sav")
+set_pass1_restoreFile("${AUTOSAVE_FILENAME}.sav")
+
+# Specify directories in which to to search for included request files
+set_requestfile_path("$(TOP)/iocBoot/$(IOC)", "")
+set_requestfile_path("$(TOP)/iocBoot/$(IOC)", "autosave")
+set_requestfile_path("$(TOP)", "BPMRFFEApp/Db")
```

Comment:
Why these many?

Reply:
I got the template from the previous IOC and forgot to update it. May I simply add `BPMRFFE/Db` path?

In this case, is it necessary to add the `asApp/Db` as well? The only request file I'm explicitly specifying is `bpmrffe.req`. Does any module attempt to read `autosave` request files?